src/widgets/AppWindow.js

```diff
@@ -19,7 +19,6 @@ import SingleLineTextInput from './SingleLineTextInput';
 import MultiLineTextInput from './MultiLineTextInput';
 import CheckBox from './CheckBox';
 import Search from './Search';
-import Radio from './Radio';
```

Comment:
No longer needs to be imported due to the revised tree grid test case.  The form test case still includes testing the radio role, but that is through the RadioGroup widget.